### DISCUSSION TITLE
Contributions calendar not working

### DISCUSSION THREAD

**Discussion by Honzoraptor31415** [community/community#114574] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

So there's this bug that I get on every user's profile page on github: when I scroll a little down to the contributions calendar and try to click on a specific day to see what contributions they made, nothing happens. I've seen this bug everytime I go to github since cca February.
The weird thing is that the calendar does work on my phone, but not on my computer.

**Comment by codejovz** [comment]
Is it correctly displayed when you just hover that day with the mousse? I think that it´s not clickable. The calendar makes bugs when you hover a day that is at it´s side, but not the one you are trying to see. Try to hover the exact day you are trying to watch without hovering the mouuse at another day before.

**Reply by Honzoraptor31415** [reply]
I don't mean the contributions count when hovering over that day in the calendar,
![image](https://github.com/community/community/assets/151644171/313e4491-71e2-4c44-9089-ed09d91287be)
because that works fine.
I mean this section,
![image](https://github.com/community/community/assets/151644171/75cce3f6-7afc-434c-beb9-7279b990c4f1)
that should display the contributions that the user made the specific day that I just clicked on, but doesn't display it.
But after clicking a specific day on my phone:
![image](https://github.com/community/community/assets/151644171/72770a46-fd8a-4a71-b9be-d56229a10d85)
It does show contributions that the user made **that specific day**.

**Comment by ethanwater** [comment]
This isn't a bug. The reason why clicking on the phone works is because there is no "hover" effect like there is on a desktop when using a mouse. The same applies to any device that is primarily touch screen.  When you hover over a day with your mouse, you see the amount of contributions on the focused day- and on touch screen devices you have to tap the day to see the same information.

Hopefully that clears up any confusion for you!

**Reply by Honzoraptor31415** [reply]
I don't mean the counter that shows up when you hover,

![image](https://github.com/community/community/assets/151644171/313e4491-71e2-4c44-9089-ed09d91287be)

because it works fine.
I mean this section,

![image](https://github.com/community/community/assets/151644171/75cce3f6-7afc-434c-beb9-7279b990c4f1)


that should display the contributions that the user made the specific day that I just clicked on, but doesn't display it.

But after clicking a specific day on my phone:

![image](https://github.com/community/community/assets/151644171/72770a46-fd8a-4a71-b9be-d56229a10d85)

It does show contributions that the user made **that specific day**.
